setup.py

```diff
@@ -18,7 +18,7 @@
 
 setup(
     name='spotipy',
-    version='2.21.0',
+    version='2.23.0',
```

Comment:
Why 2.23.0? Please keep it as 2.21.0 for now

Reply:
updated, thank you!